utils/node_local_test.py

```diff
@@ -886,6 +886,8 @@ def get_cmd_prefix(self):
         else:
             cmd.append('--leak-check=no')
 
+        cmd.append('--gen-suppressions=all')
```

Comment:
This needs to be removed before landing, right? I wonder if we could add a flag to NLT to have it optionally run in this mode, might be handy.

Reply:
This works quite well actually, the suppressions even render in Jenkins correctly.  I thought we'd added it a few months ago so I've got no problem leaving this in.